Assets/Scripts/FrogMovement.cs

```diff
@@ -39,7 +39,7 @@ void Update()
     IEnumerator FrogJump()
     {
         HasJumped = true;
-        yield return new WaitForSecondsRealtime(3f);
+        yield return new WaitForSeconds(3f);
```

Comment:
Well, it is pretty self-explanatory what is happening here, but if you decide to use this wait time in other instances, it may be worth making into it's own variable. This goes for the floats in the other classes as well.

Reply:
I am going to merge the branch, but I will leave deleting it up to you.